### PR TITLE
i18n(zh-cn): update error-reference

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

update error-reference

#### Related issues & labels (optional)

- https://github.com/withastro/docs/pull/9081

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/get-entry-deprecation-error.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Invalid use of getDataEntryById or getEntryBySlug function.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **GetEntryDeprecationError**: `METHOD` 函数已弃用，不能用于查询 "COLLECTION" 集合。请使用 `getEntry`。
+
+## 哪里出了问题？
+
+`getDataEntryById` 和 `getEntryBySlug` 函数已弃用，不能用于查询内容层集合。请使用 `getEntry` 函数。
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/errors/get-entry-deprecation-error.mdx:11]:
```suggestion
> **GetEntryDeprecationError**: `METHOD` 函数已弃用，不能用于查询 "COLLECTION" 集合。请改用 `getEntry`。

## 哪里出了问题？

`getDataEntryById` 和 `getEntryBySlug` 函数已弃用，不能用于查询内容层集合。请改用 `getEntry` 函数。
```

**Review by Nin3lee — CHANGES_REQUESTED** [review]
我把 instead 的意思翻译了进来。

**Review by Nin3lee — APPROVED** [review]
LGTM~

**Review by yanthomasdev — APPROVED** [review]
LGTM! Amazing work!

**Pull request merged**